package.json

```diff
@@ -14,6 +14,7 @@
   "devDependencies": {
     "chai": "^2.1.1",
     "clang-format": "1.0.30",
+    "closure-compiler": "mprobst/node-closure#void-args",
```

Comment:
looks like this can use 0.12 https://github.com/tim-smart/node-closure/commit/9be478c63e4923010d4c12ca4b9e1975b9c2b639